src/core/components/hv-screen/index.js

```diff
@@ -241,12 +241,44 @@ export default class HvScreen extends React.Component {
    * @param opt_href: Optional string href to use when reloading the screen. If not provided,
    * the screen's current URL will be used.
    */
-  reload = (optHref) => {
+  reload = (optHref, opts) => {
     const url = (optHref === undefined || optHref === '#')
       ? this.state.url // eslint-disable-line react/no-access-state-in-setstate
       : UrlService.getUrlFromHref(optHref, this.state.url); // eslint-disable-line react/no-access-state-in-setstate
+
+    if (!url){
```

Comment:
curious why the linter isn't capturing this
```suggestion
    if (!url) {
```